app/policies/assignment_policy.rb

```diff
@@ -103,4 +103,25 @@ def start_timed_assignment?
         check?(:start_timed_assignment?, role.accepted_grouping_for(record.id), with: GroupingPolicy))
     )
   end
+
+  def starter_file?
```

Comment:
This is generally good, but you should make use of *aliasing* of policies to reduce the code duplication. It's sufficient to define one policy called `read_starter_files?` ("read" being a conventional verb here) and then alias the others to it. This also eliminates the need for you to define a separate `see_starter_files?` policy